pkg/component/logging/eventlogger/event_logger.go

```diff
@@ -133,29 +136,10 @@ func (l *eventLogger) reconcileRBACForShoot(ctx context.Context) error {
 	}
 
 	var (
-		eventLoggerSystemRoleBinding = &rbacv1.RoleBinding{
+		eventLoggerClusterRoleBinding = &rbacv1.ClusterRoleBinding{
```

Comment:
Why? A `ClusterRoleBinding` will rather provide it with more permissions (beyond `kube-system` and `default`) instead of the opposite. What do you mean?